# multi_step walking trajectory parameters
atlas_msgs/AtlasBehaviorStepData[4] step_data
bool use_demo_walk
